dapr/proto/runtime/v1/dapr.proto

```diff
@@ -876,46 +877,50 @@ message SubtleVerifyAlpha1Response {
   bool valid = 1;
 }
 
-message WorkflowReference {
-  string instance_id = 1 [json_name = "instanceId"];
-}
-
 message GetWorkflowRequest {
-  string instance_id  = 1 [json_name = "instanceId"];
+  string instance_id  = 1 [json_name = "instanceID"];
   string workflow_component = 2 [json_name = "workflowComponent"];
-  string workflow_name = 3 [json_name = "workflowName"];
 }
 
 message GetWorkflowResponse {
-  string instance_id = 1;
-  int64 start_time = 2;
-  map<string, string> metadata = 3;
+  string instance_id = 1 [json_name = "instanceID"];
+  string workflow_name = 2 [json_name = "workflowName"];
+  google.protobuf.Timestamp start_time = 3 [json_name = "createdAt"];
+  google.protobuf.Timestamp last_updated_time = 4 [json_name = "lastUpdatedAt"];
+  string runtime_status = 5 [json_name = "runtimeStatus"];
+  map<string, string> properties = 6;
 }
 
 message StartWorkflowRequest {
-  string instance_id = 1 [json_name = "instanceId"];
+  string instance_id = 1 [json_name = "instanceID"];
   string workflow_component = 2 [json_name = "workflowComponent"];
   string workflow_name = 3 [json_name = "workflowName"];
   map<string, string> options = 4;
   bytes input = 5;
 }
 
-message WorkflowActivityRequest {
-  string instance_id = 1 [json_name = "instanceId"];
+message StartWorkflowResponse {
+  string instance_id = 1 [json_name = "instanceID"];
+}
+
+message TerminateWorkflowRequest {
+  string instance_id = 1 [json_name = "instanceID"];
   string workflow_component = 2 [json_name = "workflowComponent"];
 }
 
-message WorkflowActivityResponse {
-  // Empty
+message PauseWorkflowRequest {
+  string instance_id = 1 [json_name = "instanceID"];
+  string workflow_component = 2 [json_name = "workflowComponent"];
+}
+
+message ResumeWorkflowRequest {
```

Comment:
As the attributes for Terminate, Pause and Resume Request seems to be same, should we not keep a common request?

Reply:
No, I want to better future proof these APIs by giving each their own request message. That way we can evolve them independently over time.